content/asp.net/getting-started/controllers-actions.md

```diff
@@ -1,20 +1,122 @@
 # Understanding Controllers and Actions
-by [Steve Smith](http://deviq.com/me/steve-smith)
+by [James Chambers](http://jameschambers.com)
 
 #### Sample Files
 Download a ZIP containing this tutorial's sample files:
-- [Initial Version] - Use this as a starting point when following along with the tutorial yourself
-- [Completed Version] - Includes the completed versions of all samples
+- [Completed Version (Pending)] - Includes the completed versions of all MVC tutorials
 
-See the [Issue](https://github.com/dotnet/training-tutorials/issues/60) to claim this lesson and/or view what should be included in it.
+## Controllers and Actions in ASP.NET Core
+As we talked about in the [last tutorial](mvc.md), a _controller_ is where we put the logic needed to respond to incoming requests. Each incoming request is evaluated and mapped into a specific action on a controller by virtue of its HTTP verb, path and querystring through a process called [routing](routing.md).
 
-## First Header
+Controllers are classes that typically inherit from `Microsoft.AspNetCore.Mvc.Controller` and reside in a folder called `Controllers` in the root of your project.  Controllers are used to group together a set of related concerns, usually associated to each other by a business concept or entity.  These concerns might be related to displaying or updating data and are implemented in methods called _actions_.  
 
-Start the lesson here.
+Actions are responsible for ensuring that incoming requests are valid and that appropriate responses are returned to the client, typically building up and sending a model to a [view](views.md) for rendering. You can put whatever logic is required in an action to respond accordingly to requests, but the purpose of controllers and actions is to help separate view rendering from business logic, not to provide a place to put your business logic. Business logic should be located in services that are passed into your controller using [dependency injection](controller-dependencies.md). 
 
-...
+Each action has a role in constructing a proper HTTP response to the client, but you don't have to go it alone; the Framework has a number of features that will help you do so. You can start to  explore those features by creating a new class in the controllers folder called `PersonController`. Update the class so that it appears as below:
```

Comment:
"Go it alone" is a common phrase here...may be a colloquialism though; I'll change it.